PocketKit/Sources/PocketKit/Premium/View/PremiumUpgradeView.swift

```diff
@@ -2,7 +2,8 @@ import SwiftUI
 import Textile
 
 struct PremiumUpgradeView: View {
-    static let shouldAllowUpgrade = false
+    // TODO: remove this property and the two @State properties once we are ready to ship premium upgrades to beta users
+    static let shouldAllowUpgrade = true
```

Comment:
Will revert this to `false` before merging, so beta testers will still see the "Coming Soon!" alert